blocks/results-list-block/features/results-list/default.jsx

```diff
@@ -25,6 +25,8 @@ const ResultsList = ({ customFields }) => {
     isAdmin,
   } = useFusionContext();
   const {
+    overline,
+    overlineURL,
```

Comment:
These will never be present on customFields now it's been removed

Reply:
Yep, I missed removing these